<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditGoodDialog</class>
 <widget class="QDialog" name="EditGoodDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редагування товару</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/edit.ico</normaloff>:/icons/icons/edit.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0">
    <widget class="QLabel" name="pricePDVLabel">
     <property name="text">
      <string>Ціна з ПДВ</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="countLabel">
     <property name="text">
      <string>Кількість</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QDoubleSpinBox" name="priceSpinBox">
     <property name="maximum">
      <double>100000000000000000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="priceLabel">
     <property name="text">
      <string>Ціна</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QSpinBox" name="countSpinBox">
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="codLabel">
     <property name="text">
      <string>Артикул</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="providerLabel">
     <property name="text">
      <string>Постачальник</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="namelabel">
     <property name="text">
      <string>Назва</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="codEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QComboBox" name="providerCombo">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="text">
        <string>Редагувати</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Відхилити</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <widget class="QDoubleSpinBox" name="pricePDVSpinBox">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <double>99699999999.990005493164063</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
